<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>end</class>
 <widget class="QWidget" name="end">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>408</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>601</width>
    <height>408</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>601</width>
    <height>408</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Distance Traveled : </string>
   </property>
  </widget>
  <widget class="QLabel" name="distTraveled">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Money Spent :</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalSpent">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="goAgain">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Again</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="wineryList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>261</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="wineList">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>261</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
